x-pack/plugins/enterprise_search/public/applications/analytics/components/analytics_collection_view/analytics_collection_integrate.tsx

```diff
@@ -26,86 +29,452 @@ interface AnalyticsCollectionIntegrateProps {
   collection: AnalyticsCollection;
 }
 
+type TabKey = 'javascriptEmbed' | 'searchuiEmbed' | 'javascriptClientEmbed';
+
 export const AnalyticsCollectionIntegrate: React.FC<AnalyticsCollectionIntegrateProps> = ({
   collection,
 }) => {
   const analyticsDNSUrl = getEnterpriseSearchUrl(`/analytics/api/collections/${collection.id}`);
-  const credentials = [
+  const webClientSrc = getEnterpriseSearchUrl('/analytics.js');
+
+  const [selectedTab, setSelectedTab] = React.useState<TabKey>('javascriptEmbed');
+
+  const Steps: Record<TabKey, Array<{ title: string; children: React.ReactElement }>> = {
+    javascriptEmbed: [
+      {
+        title: i18n.translate(
+          'xpack.enterpriseSearch.analytics.collections.collectionsView.integrateTab.javascriptEmbed.stepOne.title',
+          {
+            defaultMessage: 'Embed onto site',
+          }
+        ),
+        children: (
+          <>
+            <EuiText grow={false}>
+              <p>
+                {i18n.translate(
+                  'xpack.enterpriseSearch.analytics.collections.collectionsView.integrateTab.javascriptEmbed.stepOne.description',
+                  {
+                    defaultMessage:
+                      'Embed the behavioral analytics JavaScript snippet on every page of the website or application you’d like to track.',
+                  }
+                )}
+              </p>
+              <EuiCodeBlock language="html" isCopyable>
+                {`<script src="${webClientSrc}" data-dsn="${analyticsDNSUrl}"></script>`}
+              </EuiCodeBlock>
+            </EuiText>
+          </>
+        ),
+      },
+      {
+        title: i18n.translate(
+          'xpack.enterpriseSearch.analytics.collections.collectionsView.integrateTab.javascriptEmbed.stepTwo.title',
+          {
+            defaultMessage: 'Initialise the client',
```

Comment:
```suggestion
            defaultMessage: 'Initialize the client',
```

I think US spelling is de facto standard, so just recommend changing any instances :)

Reply:
😢 Thanks for the spot!

Reply:
My autocorrect always tries to coerce me to use UK English spellings too 😃